Owlcat/bin/flag-ms.py

```diff
@@ -531,6 +535,25 @@ def parse_subset_options(options):
                         nvis_A, nvis_A * percent))
                 print(
                     "===>   %-29s includes %10d visibilities (%.3g%% of selection)" % (label, nv, nv * percent))
+                    
+            # now check for stats over CASA flagversions
+            flagvers = [tab for tab in glob.glob(f"{msname}.flagversions/flags.*") if Owlcat.tableexists(tab)]
+            if flagvers:
+                print(f"Stats for CASA-style flagversions:")
+                for vers in flagvers:
+                    name = os.path.basename(vers).split('.', 1)[1]
+                    ftab = Owlcat.table(vers, ack=False)
+                    flag_row = ftab.getcol("FLAG_ROW")
+                    flag_col = ftab.getcol("FLAG")
+                    ftab.close()
+                    flag_col[flag_row,...] = True
+                    ratio = flag_col.sum() / float(flag_col.size)
```

Comment:
needs to sum in np.uint64 precision to avoid possible overflow in signed int32 for say a 12hr 32k full pol observation